File: services/BillingMailer/DataTransferObjects/MailingDTO.cs
using System;


namespace DataTransferObjects
{
    public class MailingDTO
    {
        public int id;
        public String businessPartnerCode;
        public String businessPartnerName;
        public int codigoContrato;
        public int codigoSubContrato;
        public int diaFaturamento;
        public String destinatarios;
        public Boolean enviarDemonstrativo;
        public DateTime ultimoEnvio;


        public MailingDTO()
        {
        }
    }

}

File: DataTransferObjects/MailingDTO.php
<?php

class MailingDTO{
    var $id                   = 0;
    var $businessPartnerCode  = "";
    var $businessPartnerName  = "";
    var $contrato_id          = 0;
    var $subContrato_id       = 0;
    var $diaFaturamento       = 0;
    var $destinatarios        = "";
    var $enviarDemonstrativo  = false;
    var $ultimoEnvio          = NULL;


    function MailingDTO(){

    }

}

?>
